### ISSUE TITLE
Pagination component: underlying HTML changes for a11y

### ISSUE THREAD

**Issue by jbadan** [SAP/fundamental-styles#107]
Pagination component: https://sap.github.io/fundamental/components/pagination.html

 `<a>` tags should be `<button>`

**Comment by mikerodonnell89** [comment]
@jbadan Anchor is the correct element: https://a11y-style-guide.com/style-guide/section-navigation.html#kssref-navigation-pagination

**Comment by droshev** [comment]
We will close it for now and reopen if the issue emerges again
